docs/api/cloudlink/intro.md

```diff
@@ -8,4 +8,3 @@
     - The general request/response shape
 -->
 
-This section is out of date.
```

Comment:
This section is still out of date

Reply:
...which makes the rest of the added Cloudlink sections a little confusing as those are *not* out of date

Reply:
What about this section is incomplete?

Reply:
That would be misleading, information that is on here would contradict itself right now

Reply:
I added a list of the up to date sections, does that work?

Reply:
I added a list of the up to date sections, does that work?